src/devices/Amg88xx/Amg88xx.cs

```diff
@@ -0,0 +1,475 @@
+                {
+                    byte tl = buffer[idx++];
+                    byte th = buffer[idx++];
+                    image[c, r] = Amg88xxUtils.ConvertToTemperature(tl, th);
+                }
+            }
+
+            return image;
+        }
+
+        /// <summary>
+        /// Gets the current raw thermal image from the sensor.
+        /// </summary>
+        /// <returns>Thermal image in two's complement representation</returns>
+        public int[,] GetThermalRawImage()
+        {
+            // the readout process gets triggered by writing to pixel 0 of the sensor w/o any additional data
+            _i2cDevice.WriteByte((byte)Register.T01L);
+
+            Span<byte> buffer = stackalloc byte[Rows * Columns * BytesPerPixel];
+            _i2cDevice.Read(buffer);
+
+            var image = new int[Columns, Rows];
+
+            int idx = 0;
+            for (int r = 0; r < Rows; r++)
+            {
+                for (int c = 0; c < Columns; c++)
+                {
+                    byte tl = buffer[idx++];
+                    byte th = buffer[idx++];
+                    image[c, r] = th << 8 | tl;
+                }
+            }
+
+            return image;
+        }
+
+        /// <summary>
+        /// Gets the temperature reading from the internal thermistor.
+        /// </summary>
+        /// <returns>Temperature reading</returns>
+        public Temperature GetSensorTemperature()
+        {
+            byte tthl = GetRegister(Register.TTHL);
+            byte tthh = GetRegister(Register.TTHH);
+
+            int reading = (tthh & 0x7) << 8 | tthl;
+            reading = tthh >> 3 == 0 ? reading : -reading;
+
+            // The LSB is equivalent to 0.0625℃.
+            return Temperature.FromDegreesCelsius(reading * ThermistorTemperatureResolution);
+        }
+
+        #endregion
+
+        #region Status
+
+        /// <summary>
+        /// Gets the temperature overflow flag from the status register
+        /// </summary>
+        /// <returns>Temperature overflow flag</returns>
+        public bool HasTemperatureOverflow()
+        {
+            return GetBit(Register.STAT, (byte)StatusFlagBit.OVF_IRS);
+        }
+
+        /// <summary>
+        /// Clears the temperature overflow flag in the status register
+        /// </summary>
+        public void ClearTemperatureOverflow()
+        {
+            // only the bit to be cleared is set, the other bits need to be 0
+            SetRegister(Register.SCLR, 1 << (byte)StatusClearBit.OVFCLR);
+        }
+
+        /// <summary>
+        /// Gets the thermistor overflow flag from the status register.
+        /// Note: the bit is only menthioned in early versions of the reference specification.
+        /// It is not clear whether this is a specification error or a change in a newer
+        /// revision of the sensor.
+        /// </summary>
+        /// <returns>Thermistor overflow flag</returns>
+        public bool HasThermistorOverflow()
```

Comment:
how is this different from TemperatureOverflow? Can you describe the docs more? Also consider collapsing this and next method into one

Reply:
@krwq Same as for the `HasTemperatureOverflow` method... I don't know much about its use in applications. I can only tell that this is a kind of error indicator which needs to be reset manually. An automatic reset doesn't make sense to me. 
I'll extend the documentation accordingly.